rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -984,7 +984,7 @@ int LuaUnsyncedRead::GetMiniMapGeometry(lua_State* L)
 /*** Get minimap rotation
  *
  * @function Spring.GetMiniMapRotation
- * @return number amount in radians
+ * @treturn number amount in iteration of 90 degrees (0, 1, 2, 3)
```

Comment:
LDoc `treturn` is no longer supported, pls use LLS/EmmyLua `@return`

That is: `@return <type> [name] [description...]`

This should be:

```c
/*** Get minimap rotation
 *
 * @function Spring.GetMiniMapRotation
 * @return MiniMapRotation rotation
 */
```

If you do did not define a `MiniMapRotation` alias, then it would be:

```c
 * @return 0|1|2|3 rotation Amount in iterations of 90 degrees.
```

Would be worth saying if it's CW or CCW though imo.